.env.example

```diff
@@ -46,4 +46,6 @@ EXCHANGE_WALLET_PRIV=0xd9066ff9f753a1898709b568119055660a77d9aae4d7a4ad677b8fb3d
 ISSUER_ID=Issuer ID
 
 # Possible options: GRID_OPERATOR and LOCATION, should be comma separated
-DEVICE_PROPERTIES_ENABLED=GRID_OPERATOR,LOCATION
\ No newline at end of file
+DEVICE_PROPERTIES_ENABLED=GRID_OPERATOR,LOCATION
+# Amount of energy in single certificate 
+ENERGY_PER_UNIT=1000000
```

Comment:
Should this be a global setting and used in the rest of the packages? E.g. in the UI.